model/src/main/kotlin/config/DownloaderConfiguration.kt

```diff
@@ -20,9 +20,16 @@
 package org.ossreviewtoolkit.model.config
 
 import org.ossreviewtoolkit.model.SourceCodeOrigin
+import org.ossreviewtoolkit.model.licenses.LicenseCategory
 import org.ossreviewtoolkit.spdx.getDuplicates
 
 data class DownloaderConfiguration(
+    /**
+     * Configuration which packages with licenses from the [license categories][LicenseCategory] should be considered
```

Comment:
Maybe "Configuration *about* which" or "Configuration *for* which"? Somehow a word is missing here for my understanding.

Reply:
Hmm, maybe it's simpler to find a wording that does not include "configuration" as that's part of the class name anyway (anything in here is configuration). Like:

> The categories licenses of packages need to be part of in order to get included into the download.

And while at it, maybe the docs for `sourceCodeOrigins` could be reworded accordingly (in a new commit of course)?